src/components/Game.js

```diff
@@ -340,12 +357,9 @@ function Game() {
               />
             </div>
             <div>
-              <img
-                height="14px"
-                style={{ marginTop: "5px" }}
-                src={require(`../assets/pt2.png`)}
-                alt="pt2"
-              />
+                <p style={{marginTop:'5px', color:'yellow', fontFamily:'fantasy', fontSize:'13px', fontWeight:'600'}}>
+                {Player2.student_name === undefined?"PLAYER 2":Player2.student_name}
```

Comment:
use OR condition

Reply:
for what

Reply:
i player2 image instead of i added Text 
and if I will get  Player2.student_name === undefined then I am showing that text instead of image else showing real username